java-security/src/main/java/com/sap/cloud/security/x509/CfX509CertExtractor.java

```diff
@@ -0,0 +1,26 @@
+package com.sap.cloud.security.x509;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.Nullable;
+import javax.servlet.http.HttpServletRequest;
+
+import static com.sap.cloud.security.x509.X509Constants.FWD_CLIENT_CERT_HEADER;
+
+/**
+ * X509 certificate accessor implementation for Cloud Foundry environment.
+ */
+public class CfX509CertExtractor implements X509CertExtractor {
```

Comment:
It should be the same for K8s

Reply:
but there's a difference how x-forwarded-cert can be delivered in k8s env, that's why 2 implementations